### ISSUE TITLE
Exception: Can't get attribute 'DetectionModel' on <module 'models.yolo'

### ISSUE THREAD

**Issue by L0calBoy** [ultralytics/yolov5#9191]
### Search before asking

- [X] I have searched the YOLOv5 [issues](https://github.com/ultralytics/yolov5/issues) and found no similar bug report.

### Bug

Hi im using yolov5 this week and can´t get a new custom data every time i train a custom data and i put in my code i get this error 

"Exception: Can't get attribute 'DetectionModel' on <module 'models.yolo' from 'C:\\Users\\pc/.cache\\torch\\hub\\ultralytics_yolov5_master\\models\\yolo.py'>. Cache may be out of date, try `force_reload=True` or see https://github.com/ultralytics/yolov5/issues/36 for 
help." 

i try using other labels / less pictures / more pictures and idk what to do pls help :(

this is my code 

### Environment

YOLOV5 6.2 

### Minimal Reproducible Example

#importar libreriaaas
#pip install opencv-python

#SEPARAR LOS IMPORTS PARA HACERLOS EN .BAT
#CREAR UAN CARPETA CON LOS REQUERIMIENTO

from asyncio.windows_events import NULL
import datetime
from pickle import FALSE
from time import time
from charset_normalizer import detect
import torch
import cv2
#import utils
import pyautogui
import numpy as np
#from numpy import imag
import pywhatkit as ws
import pandas
import os
ahora = datetime.datetime.now()
deteccion = 0
# Leeer el modelo 
model = torch.hub.load('ultralytics/yolov5','custom',path='C:/Users/pc/Desktop/Vidio/model/furritas.pt') #aqui hay q cambiar
cap = cv2.VideoCapture(1)

anexo = 0
cap = cv2.VideoCapture(1,cv2.CAP_DSHOW)
image_path = r'CAPTURAS/captura.jpg'
direccion = r'CAPTURAS'
img = cv2.imread(image_path)
imagen ='C:/Users/pc/Desktop/Vidio/CAPTURAS/captura.jpg'
nombre = 'captura.jpg'
os.chdir(direccion) 
texto1 = "Sé ha detectado un intruso armado!, Tomar las precauciones necesarias captura tomada"
texto2 = ahora.strftime('%d/%m/%Y  %H:%M')
texto3 = texto1+texto2
timelapse=200

def enviaralerta(imagen):

    try:
        ws.sendwhats_image('',imagen,texto3,15,True,5)

    except:
        pass 

#start
while True:

    #print(imagen)

    #Realizar lectura del papu
    ret, frame = cap.read()
    #deteccion del papu
    detect = model(frame)
    info = detect.pandas().xyxy[0]
    #print(info.confidence)
    
    if (info.size > 0 and timelapse >= 200):
        deteccion = info.confidence[0]
    else:
        pass #print("Todo piola")

    #Mostratr fps
    #print(deteccion)
    cv2.imshow('Detector de armas',np.squeeze(detect.render())) 
    #print(texto3)

    timelapse = timelapse+1

    if timelapse > 9999:
        timelapse = 200

    #print(timelapse)

    if (deteccion >= 0.65):
        
        #print(texto3)       
        
        cv2.imwrite('captura.jpg',np.squeeze(detect.render()))        
        #cv2.destroyAllWindows()
        #time out para que deje hacerlo del wassap         
        #condicion de cierre
        
        #enviaralerta(imagen)
        pyautogui.click()

        timelapse=0
        deteccion=0

    t = cv2.waitKey(5)
    if t == 27:
        break

cap.relase()
cv2.destroyAllWindows()

### Are you willing to submit a PR?

- [X] Yes I'd like to help by submitting a PR!

**Comment by glenn-jocher** [comment]
@L0calBoy use force_reload=True

**Comment by L0calBoy** [comment]
I LOVE YOU THAT WORKS U SAVE ME MAN <3

**Comment by glenn-jocher** [comment]
@L0calBoy glad to hear that it worked for you! If you have any more questions or need further assistance, feel free to ask. The YOLOv5 community and the Ultralytics team are always here to help!
